icu4j/main/tests/core/src/com/ibm/icu/dev/test/util/ULocaleTest.java

```diff
@@ -1001,13 +1001,16 @@ public void TestCanonicalization(){
     }
 
     @Test
-    public void TestGetAvailable(){
+    public void TestGetAvailable() {
         ULocale[] locales = ULocale.getAvailableLocales();
-        if(locales.length<10){
-            errln("Did not get the correct result from getAvailableLocales");
+        if (locales.length < 1) {
+            errln("Did not get any results from getAvailableLocales");
         }
-        if(!locales[locales.length-1].getName().equals("zu_ZA")){
-            errln("Did not get the expected result");
+        for (ULocale locale : locales) {
+            int length = locale.getLanguage().length();
```

Comment:
I'd feel more confident if you also checked for a well-formed language; i.e., all lowercase ascii letters.

Reply:
Done